### PR TITLE
Minor shield updates.jan2023

### PR DESCRIPTION
**Pull request by Diggles27**
Meow helped me pull Battania shield from Multiplayer assests

### CODE REVIEW

**Comment by AngryMob010** [comment]
rebased
re-exported with v23.

needs stat adjustments before merge

File: items.json [hunk 1]
```diff
@@ -13486,6 +13486,44 @@
...
+          "HasHitPoints",
+          "CanBlockRanged"
+        ],
+        "thrustDamage": 5,
+        "thrustDamageType": "Blunt",
+        "thrustSpeed": 55,
+        "swingDamage": 0,
+        "swingDamageType": "Blunt",
+        "swingSpeed": 55
+      }
```
Comment (namidaka) [inline items.json:13524]: Seems like Length does nothing , see with meow

**Review by namidaka — CHANGES_REQUESTED** [review]

File: items.json (hunk 1 above)
Comment (AngryMob010) [reply items.json:13524]: the shield has a physically larger model i believe that's why Diggles wanted to add it

**Comment by AngryMob010** [comment]
reverted buckler stat changes - will be changed at a later time

**Review by namidaka — APPROVED** [review]

**Pull request merged**